tests/test_edgeql_update.py

```diff
@@ -2937,10 +2939,10 @@ async def test_edgeql_update_add_dupes_03(self):
                     UPDATE UpdateTestSubSubType
                     FILTER .name = 'add-dupes'
                     SET {
-                        tags += {
+                        tags += assert_distinct({
                             (UPDATE Tag FILTER .name = 'fun' SET {flag := 1}),
                             (UPDATE Tag FILTER .name = 'wow' SET {flag := 2}),
-                        }
+                        })
```

Comment:
Do we actually *need* an `assert_distinct` here? I thought that `+=` and `-=` already will make linked sets distinct in the end. I mean if I already have `Tag` "wow" linked as `tags`, doing `tags += (select Tag filter .name = 'wow')` should be legal, so who cares if the set after `+=` or `-=` is not distinct if the result must be reconciled anyways?

Reply:
This is a good point.  I think we can disable multiplicity checks for `-=` and `+=`

Reply:
+1